cypress/support/pages/admin/pageAdministration.js

```diff
@@ -33,6 +33,7 @@ class Management {
     static #teamAdministrationNavigationButton = '[data-testid="Teams"]'
     static #schoolAdministrationNavigationButton = '[data-testid="Schule"]'
     static #studentTeamCheckbox = '[data-testid="student_team_checkbox"]'
+    static #LearnstoreCheckbox = '[id="studentlernstorevisibility"]'
```

Comment:
1. it should start with small letter `learnstoreCheckbox`. Also change the name as well. because its confusing that this checkbox is only for student or other roles have sharing it?
2. IMO it should be `learningstore` rather than `learnstore`. As you have class named as `Learning_Store` later in the method implementation file